web/impact/impact/v1/views/user_detail_view.py

```diff
@@ -1,46 +1,63 @@
 from django.contrib.auth import get_user_model
 from rest_framework.response import Response
-from rest_framework.views import APIView
-from rest_framework_tracking.mixins import LoggingMixin
 
 from impact.permissions import (
     V1APIPermissions,
 )
 from impact.v1.helpers import (
+    USER_FIELDS,
     UserHelper,
     valid_keys_note,
 )
 from impact.v1.metadata import ImpactMetadata
+from impact.v1.views import ImpactView
+
 
 INVALID_KEYS_ERROR = "Recevied invalid key(s): {invalid_keys}."
 NO_USER_ERROR = "Unable to find user with an id of {}"
 
 User = get_user_model()
 
 
-class UserDetailView(LoggingMixin, APIView):
+class UserDetailView(ImpactView):
```

Comment:
why were all the LoggingMixins removed? this is required for rest-framework-tracking (logging requests to django admin)

Reply:
ImpactView inherits from LoggingMixin and APIView.